imageroot/bin/module-cleanup-state

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
Is this script really used?
I see the same `rm` commands inside other scripts...